rollup/plugins/bundlePlugins.js

```diff
@@ -187,8 +187,12 @@ export function makeTSPlugin(jsVersion) {
  * from the browser and browser+tracing bundles.
```

Comment:
l: Let's just update the JSDoc here that this factory function can be used to exclude basically any exported member from the bundles